en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
-        
-        ![Configure HTTP Connection](../assets/img/services/select-existing-connection.png)
-        
-        | **Field**               | **Value**                      |
-        |-------------------------|--------------------------------|
-        | **OPERATION**           | **get**                        |
-        | **Resource Path**       | `"/total/country/united-states"` |
-        | **Select Payload Type** | **JSON**                       |
+    5. Enter the following information:
         
+        | **Field**               | **Value**                       |
+        |-------------------------|---------------------------------|
+        | **OPERATION**           | **get**                         |
+        | **Resource Path**       | `"/total/country/united-states"`|
+        | **Select Payload Type** | **JSON**                        |
+
         !!! info
             To select **JSON** as the payload type, you need to click **Do you want to extract a payload?**.
             ![Do you want to extract a payload button](../assets/img/services/enable-payload.png)
-        
-        Click **Save & Done**.
-        
-        Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
+            
+    6. Click **Save & Done**.
     
-7. To cast all the data retrieved via the API connected to the service before further processing, add a new custom statement as follows:
+       Now your API trigger is connected to the public API that fetches the active COVID-19 cases in the selected country.
+
+7. Add a new custom statement as follows to cast all the data retrieved via the API connected to the service before further processing:
 
     1. Click the last **+** icon in your low-code diagram.
 
         ![Add New Statement](../assets/img/services/add-custom-statement.png)
     
-    2. Click **Other** and then enter the following in the **Statement** field.
-    
-        ```
+    2. Click **Other** and then enter the following in the **Statement** field:
+
+        ```ballerina
         json[] jsonArray = <json[]>jsonPayload;
         json[] response = [];
         ```
-        Then click **Save**.
-    
-8. To filter the dates on which the active cases have exceeded 5,000, let's add a statement of the `ForEach` type as follows: 
+    3. Click **Save**.
+
+8. Add a ForEach statement as follows to filter the dates where the active cases have exceeded 5,000:
 
     1. Click the last **+** icon in your low-code diagram.
     
     2. Click **ForEach**.
 
     3. In the **Iterable Expression** field, enter `jsonArray`. Then click **Save**.
     
-    4. To apply the filter mentioned, add a custom statement within the ForEach statement. To do this, click the **+** icon just below the last ForEach statement you added.
+    4. To apply the filter, add a custom statement within the ForEach statement. To do this, click the **+** icon just below the last ForEach statement you added.
```

Comment:
What's the filter?  filter the dates where the active cases have exceeded 5,000?

Reply:
Do we need to mention it again? I feel it confuses the reader. If we continue with steps I think it would be better, because this is mentioned above.

Reply:
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108